source/_includes/code/application-framework/login-page/doc.md

```diff
@@ -1,4 +1,23 @@
 <table class="pforg-code-usage-table">
+  <thead>
+    <tr>
+      <th>Selector</th>
+      <th>.login-pf</th>
+  </thead>
+  <tbody>
+  <tr>
+    <td>Applied to</td>
+    <td><code>body</code></td>
+  </tr>
+  <tr>
+    <td>Required</td>
+    <td><strong>No</strong></td>
```

Comment:
wouldn't this be required?

Reply:
I would think it would be required..?

Reply:
It is only required if they want the background image...should it be required, as that is the design pattern?

Reply:
ya that's my thought. If it's removed it's white box on a white background, which looks pretty strange.